emailer.py

```diff
@@ -82,6 +82,14 @@ def commit_email():
     pusher_email = '{0} <{1}>'.format(json_dict['pusher']['name'],
                                       json_dict['pusher']['email'])
 
+    commit_SHA = json_dict['after']
+    url = f"https://api.github.com/repos/chapel-lang/chapel/commits/{0}/pulls".format(commit_SHA)
```

Comment:
Python doesn't really have constants, but you could factor out the GitHub API URL and repos name (chapel-lang) into variables at the module level (or make an API call function or something) just in case those change and we end up using them in multiple places.

Not really a big deal for now though as it doesn't look like we duplicate them anywhere and I don't think those URLs are likely to change.

Reply:
Just pushed new code, were you thinking something like that?

Reply:
Something like that, yeah.

I'd put `GITHUB_OWNER` and `GITHUB_REPO` in all caps (unless you know that's not standard practice in Python).  I assume you need to put quotes around 'chapel-lang' and 'chapel' too, and maybe you'd need to add `global GITHUB_OWNER, GITHUB_REPO` at the start of your function to pull those variables into scope (not sure).

I assume you have some way of testing this script before deploying it?